app/assets/stylesheets/views/_covid.scss

```diff
@@ -303,6 +303,16 @@ $c19-landing-page-header-background: govuk-colour("blue");
   }
 }
 
+.covid__logos-post-19-july {
+  width: 100%;
+  max-width: 500px;
+  margin-bottom: govuk-spacing(4);
```

Comment:
I think this needs a smaller margin-top setting. The logos seem a little far from the page header compared to the design.

Reply:
Design:
<img width="835" alt="Screenshot 2021-07-16 at 14 07 31" src="https://user-images.githubusercontent.com/5793815/125952744-ea9b1ae6-9d13-4107-82e1-912e5d6f9771.png">

Actual:
<img width="835" alt="Screenshot 2021-07-16 at 14 08 02" src="https://user-images.githubusercontent.com/5793815/125952840-c33b191e-da4c-4d14-a0c1-a51a51b0830e.png">